CodeiumVS/LanguageServer/LanguageServer.cs

```diff
@@ -192,9 +192,29 @@ public async Task PrepareAsync()
         var waitDialogFactory = (IVsThreadedWaitDialogFactory)await VS.Services.GetThreadedWaitDialogAsync();
         IVsThreadedWaitDialog4 progDialog = waitDialogFactory.CreateInstance();
 
+
+
+        string extensionBaseUrl = "https://github.com/Exafunction/codeium/releases/download";
+        string languageServerVersion = Version;
+        if (Package.SettingsPage.EnterpriseMode)
+        {
+            // Get the contents of /api/extension_base_url
+            try
+            {
+                string portalUrl = Package.SettingsPage.PortalUrl.TrimEnd('/');
+                string result = await new HttpClient().GetStringAsync(portalUrl + "/api/extension_base_url");
+                extensionBaseUrl = result.Trim().TrimEnd('/');
+                languageServerVersion = await new HttpClient().GetStringAsync(portalUrl + "/api/version");
+            }
+            catch (Exception)
+            {
+                await Package.LogAsync("Failed to get extension base url");
```

Comment:
If it is not critical, we should fall back to the default language server in case it fails to get the version. If it is, maybe we should show a message box, the log doesn't get checked very often.
```suggestion
                await Package.LogAsync("Failed to get extension base url");
                extensionBaseUrl = "https://github.com/Exafunction/codeium/releases/download";
```

Reply:
it's not super critical.